.gitignore

```diff
@@ -10,8 +10,9 @@ yarn-error.log*
 .DS_Store
 .idea/
 public/
+.vscode/
 
-showcase/bundle.js
-showcase/bundle.css
+bundle.js
+bundle.css
```

Comment:
Are these exported bundles or bundles for the showcase? Should they be in. packages/showcase instead?